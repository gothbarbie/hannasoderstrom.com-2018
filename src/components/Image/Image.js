import styled from 'styled-components'

const Image = styled.img`
  width: 100%;
  height: auto;
  display: block;
  ${({ maxWidth }) =>
    maxWidth && maxWidth.length && 'max-width: ' + maxWidth + 'px'};
`

export default Image
