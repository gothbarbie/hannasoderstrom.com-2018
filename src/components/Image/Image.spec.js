import React from 'react'
import { shallow } from 'enzyme'

import Image from './index'

describe('components/Image', () => {
  let component

  beforeEach(() => {
    component = shallow(<Image />)
  })

  it('renders', () => {
    expect(component).toMatchSnapshot()
  })
})
