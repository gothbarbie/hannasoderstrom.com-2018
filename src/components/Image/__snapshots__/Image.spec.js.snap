// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/Image renders 1`] = `
<img
  className="sc-bdVaJa hLhmaB"
/>
`;
